docs/connecting-to-merlin/python-sdk.md

```diff
@@ -15,6 +15,10 @@ from merlin.model import ModelType
 # Connect to an existing Merlin deployment
 merlin.set_url("merlin.example.com")
 
+# Set the active model to the name given by parameter, if the model with the given name is not found, a new model will 
+# be created.
+merlin.set_model("example-model", ModelType.PYFUNC)
+
```

Comment:
Added this section here because trying to actually call `list_model_endpoints` without setting a model will throw an error.